.env.test

```diff
@@ -0,0 +1,2 @@
+API_KEY=t32i4893d9awq09a788a79b51k66295
+API_URL=https://api.openweathermap.org/data/2.5
```

Comment:
**Never use real credentials in tests**

and also

**Never add credentials to git history**

copy this file into a file called `.env` :

```sh
cp .env.test .env
```

then change the values in the test environment file:

```sh
# in .env.test
API_KEY=api_key
API_URL=https://api.openweathermap.org/data/2.5
```
and add `.env` to `.gitignore`